common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -688,19 +692,21 @@ describe 'Problem', ->
         $('#answer_1_1').html('One')
         $('#answer_1_2').html('Two')
 
-      it 'hide the answers', ->
-        @problem.show()
-        expect($('#answer_1_1')).toHaveHtml ''
-        expect($('#answer_1_2')).toHaveHtml ''
-        expect($('label[for="input_1_1_1"]')).not.toHaveAttr 'correct_answer'
+  describe 'displayShowAnswerTooltip', ->
+    beforeEach ->
+      @problem = new Problem($('.xblock-student_view'))
+      @problem.el.find('.show').isMouseFocus = false
```

Comment:
I'm confused how this is working. How does this line trigger the display of the tooltip?